anonymize_test.py

```diff
@@ -0,0 +1,24 @@
+from anonymize import anonymize
```

Comment:
very good test file! you can improve it by defining fixtures (https://docs.pytest.org/en/7.1.x/how-to/fixtures.html) at the beginning of your file and use them in the tests. you can define one fixture for the file `anonymized_text.txt`, and one for `test_example.pdf`.